src/components/modals/QrCodeModal.tsx

```diff
@@ -150,7 +150,7 @@ class QrCodeModalComponent extends React.Component<Props, State> {
             />
           )}
         </View>
-        <ModalCloseArrow onPress={() => bridge.resolve()} />
+        <ModalFooter fadeOut onPress={() => bridge.resolve()} />
```

Comment:
The `fadeOut` looks really bad. Please get rid of it:

![simulator_screenshot_2DD11F87-0836-4D6A-ACB5-533365C7F9D3](https://github.com/EdgeApp/edge-login-ui-rn/assets/276214/2921140f-9911-45e2-bb29-12b4aeaaa2ce)

Reply:
Demo of using refactored `<ModalScrollArea>` to attempt to resolve this issue whilst keeping the fadeout.

| QR Modal | TextInputModal |
|:--------:|:--------------:|
| ![Simulator Screen Recording - iPhone 8 - 2023-06-08 at 21 02 13](https://github.com/EdgeApp/edge-login-ui-rn/assets/4023066/dd7d8e5e-dcc8-4461-930b-720cc20f0604) | ![Simulator Screen Recording - iPhone 8 - 2023-06-08 at 20 46 40](https://github.com/EdgeApp/edge-login-ui-rn/assets/4023066/19ae9c49-e6b2-42b5-ac11-6bc3e31e2fc2) |